src/pages/docs/platform-hub/index.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: src/layouts/Default.astro
+pubDate: 2025-09-30
+modDate: 2025-09-30
+title: Platform Hub
+subtitle: An overview of Platform Hub
+icon: 
+navTitle: Overview
+navSection: Platform Hub
+description: Platform Hub is a new capability in Octopus that helps platform teams standardize how software is delivered across teams using connected templates and enforceable policies. Together, these features create a governance layer for software delivery, making it easier for platform teams to scale best practices, reduce drift, and deliver with confidence.
+navOrder: 70
```

Comment:
Should double check page and nav titles are what you want. A lot of things are saying Overview at the moment